docs/tables/azure_mysql_server.md

```diff
@@ -118,3 +118,63 @@ from
   azure_mysql_server,
   jsonb_array_elements(server_keys) as keys;
 ```
+
+### List server configuration details
```

Comment:
**Note:** `Server configurations` is the same as `Server parameters` as shown in Azure MySQL server console